src/sentry/api/endpoints/project_group_index.py

```diff
@@ -455,6 +454,10 @@ def put(self, request, project):
                             group=group,
                             reason=GroupSubscriptionReason.status_change,
                         )
+                        if not GroupAssignee.objects.filter(
+                            group=group, project=group.project
+                        ).first():
+                            result['assignedTo'] = (User.objects.filter(id=acting_user.id).first())
```

Comment:
@tkaemming my bad I wrapped it in parenthesis, removing them. I performed this query because lazyobject was not serializable (it is not present in the registry and hence it was breaking there).